rclpy/rclpy/context.py

```diff
@@ -20,6 +20,10 @@
 import weakref
 
 
+g_logging_configure_lock = threading.Lock()
+g_logging_ref_count = 0
```

Comment:
@ivanpauno same comment as in https://github.com/ros2/rclcpp/pull/998 about where this reference count lives.

Reply:
See my reply in the `rclcpp` PR.